### ISSUE TITLE
Files lose the color tag after editing

### ISSUE THREAD

**Issue by Fuziius** [files-community/Files#10186]
### Description

I have Windows 11 and the Files version 2.3.12.0. For all my Excel tables I have a color tag. Every time I edit and save one of the Excel files, the color tag is gone and I have to find and re-tag the file.

### Steps To Reproduce

1. tag (Excel) file
2. open, modify and save file
3. tag is gone

### Steps to fix

How can the files keep the tag after editing

### Files Version

Version 2.3.12.0

### Windows Version

Windows 11 Home 22H2

### Relevant Assets/Logs

none

**Comment by sthivaios** [comment]
I have tried this with a text file, and the bug doesn't happen to me.

I will try it with excel too and let you know.

**Comment by Josh-65** [comment]
You will need to upload the logs files directly to GitHub. You can get them in files settings > about > open log folder.

**Comment by sthivaios** [comment]
@Josh-65 I know you aren't asking me for the log files, but I am downloading the Lab Test Kit from the Microsoft Evaluation Center right now, to try it with excel. It does not happen with notepad though, as I said earlier. Also, if this bug actually "works" with excel, where can I upload the log files? Here?

**Comment by Josh-65** [comment]
You can just drag the two .log files into the text box and wait for it to show something like [debug.log] (generated URL)

**Comment by sthivaios** [comment]
Yeah, just like I attach videos and stuff. Ok, so just attach it to the comment. Sure, will do.

**Comment by Fuziius** [comment]
Hello @Josh-65 , I did open the log folder but what data should I upload form there? Or is it the whole folder?

**Comment by Josh-65** [comment]
There should be two files debug.log and debug_fulltrust.log

**Comment by Fuziius** [comment]
Thanks for the help. If you need something else please let me know.
[debug.log](https://github.com/files-community/Files/files/9768250/debug.log)
[debug_fulltrust.log](https://github.com/files-community/Files/files/9768251/debug_fulltrust.log)

**Comment by yaichenbaum** [comment]
My guess is Excel removes the alternate data stream that holds the tag. Pinging @hez2010 for thoughts on this issue.
